### DISCUSSION TITLE
4.5.3打不开

### DISCUSSION THREAD

**Discussion by liushihowe** [ywmoyue/biliuwp-lite#60] [Q&A]
无法打开此应用包？？？

解析应用包时出错

**Comment by liushihowe** [comment]
![image](https://user-images.githubusercontent.com/127091329/225560452-b9d61117-e2a1-4f14-b5b5-85debd11d70f.png)

**Comment by GrandDuke1106** [comment]
试试用powershell的'''Add-AppxPackage'''命令

**Comment by liushihowe** [comment]
> 试试用powershell的'''Add-AppxPackage'''命令

谢谢你，这个方法可以
